x-pack/plugins/observability/public/pages/alerts/index.tsx

```diff
@@ -44,6 +45,14 @@ export function AlertsPage({ routeParams }: AlertsPageProps) {
     query: { rangeFrom = 'now-15m', rangeTo = 'now', kuery = '', status = 'open' },
   } = routeParams;
 
+  useBreadcrumbs([
+    {
+      text: i18n.translate('xpack.observability.breadcrumbs.alertsLinkText', {
+        defaultMessage: 'Alerts',
+      }),
+    },
+  ]);
```

Comment:
This looks like a good way to set breadcrumbs. Should we do something similar for APM? 

Btw. I'm wondering if Kibana should provide something like this. Seems odd that every plugin reinvents the wheel.

Reply:
Agreed, if this can be moved up the hierarchy, we had this hook originally in uptime, and we moved it here, to be used in UX app and other places.

Reply:
I think we can use this mostly as-is in APM or other places (it's currently used in UX as Shahzad mentioned.)

There's ongoing discussion in elastic/observability-design#53.